nuxeo-java-client/src/main/java/org/nuxeo/client/NuxeoClient.java

```diff
@@ -92,6 +97,19 @@ protected void addOkHttpInterceptor(Interceptor interceptor) {
         buildRetrofit();
     }
 
+    protected String computeUserAgent() {
+        String nuxeoPart = " NuxeoJavaClient/";
+        try (InputStream inputStream = getClass().getResourceAsStream("/config.properties")) {
```

Comment:
Could we have a more specific filename for this? The client will be a library used by various applications and we don't want to pollute the root of the classpath...